src/app/auth.config.ts

```diff
@@ -1,24 +1,9 @@
-import { AuthConfig } from 'angular-oauth2-oidc';
-
-export const authConfig: AuthConfig = {
-
-  // Url of the Identity Provider
-  issuer: 'https://sso-dev.pathfinder.gov.bc.ca/auth/realms/zdw7o87p',
-
-  // URL of the SPA to redirect the user to after login
-  redirectUri: window.location.origin,
-
-  silentRefreshRedirectUri: window.location.origin + '/silent-refresh.html',
-  // The SPA's id.
-  // The SPA is registerd with this id at the auth-server
-  clientId: 'demo-app',
-
-  // set the scope for the permissions the client should request
-  // The first three are defined by OIDC.
-  scope: 'openid profile email',
-  // Remove the requirement of using Https to simplify the demo
-  // THIS SHOULD NOT BE USED IN PRODUCTION
-  // USE A CERTIFICATE FOR YOUR IDP
-  // IN PRODUCTION
-  requireHttps: false
+export const authConfig = {
+  clientID: "974b09f9-8093-476b-a795-9e6b4e317f6b",
```

Comment:
Will need to hide it if it is secret.

Reply:
as discussed: This isn't a secret, it is included in the URL when users are logging in